### ISSUE TITLE
PLY surface not showing colors

### ISSUE THREAD

**Issue by RubenChM** [nglviewer/ngl#1059]
Using NGL 2.4.0, I loaded the outname.ply from [EDTSurf](https://zhanggroup.org/EDTSurf/) using the examples/webapp.html but it only display the surface in white.

**Comment by fredludlow** [comment]
Did this used to work in older versions of NGL? (I never used PLY surfaces properly, though I helped with some of the porting recently).

Looking at the code we don't parse the color data in `PLYLoader`, though the parser is capable of doing so (it's explicitly disabled).

**Comment by RubenChM** [comment]
I don't think it worked, at least it is the same using the [2.0 from the web](http://nglviewer.org/ngl/). 

Where in the code disabled?
